students/kulon_aleksander/lesson_02/lost_card.py

```diff
@@ -1,9 +1,10 @@
 n = int(input())
-result = 0
 
+sum = 0
```

Comment:
Why did you change this name :) sum is built-in function - try to avoid built-in functions names:
https://docs.python.org/3/library/functions.html